torch/_decomp/__init__.py

```diff
@@ -100,6 +100,7 @@ def add_op_to_table(aten_op):
                 if op_overload in registry:
                     raise RuntimeError(f"duplicate registrations for {op_overload}")
                 registry[op_overload] = fn
+                op_overload.py_impl(torch._C.DispatchKey.Meta)(fn)
```

Comment:
Hmmm won't this unconditionally register a bunch of our python decomps to the meta key, and stomp over some of our C++ decomps/meta kernels?

Reply:
oh nvm is this fine because the PythonDispatcher needs to be explicitly turned on elsewhere, anyway.